pkg/node/manager/manager.go

```diff
@@ -477,6 +478,10 @@ func (m *manager) NodeUpdated(n nodeTypes.Node) {
 			continue
 		}
 
+		if addr := address.ToString(); address.Type == addressing.NodeCiliumInternalIP && m.conf.IsLocalRouterIP(addr) {
+			continue
```

Comment:
Actually, I think we want 

```suggestion
			dpUpdate = false
```

here instead to not break other behavior like inserting the IP into the ipcache.

Reply:
We still want the ipcache entry for that IP.